src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
+              <set>
+                <member>True</member>
+                <member correct="yes">False</member>
+              </set>
             </var>
             <pg-code>
+              $true=PopUp(['?','True','False'],1);
+              $false=PopUp(['?','True','False'],2);
             </pg-code>
           </setup>
           <statement>
-            <p>T/F: A study of related rates is part of the standard police officer training.</p>
+            <p>T/F: A study of related rates is part of the standard police officer training. <var name="$false" form="popup"/></p>
           </statement>
-          <solution>
-            <p>F</p>
-          </solution>
         </webwork>
       </exercise>
     </exercisegroup>
+
+<!--Group 2: Related rates answer entry-->
     <exercisegroup>
       <title>Problems</title>
+<!--Exercise 3: Puddle of depth 10 cm growing-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$a">
+              <static>0.7958</static>
+            </var>
+            <var name="$b">
+              <static>0.07958</static>
+            </var>
+            <var name="$c">
+              <static>0.007958</static>
             </var>
             <pg-code>
+              <!--FUNGUS: Is this correct?-->
+              $a = 0.7958;
+              $b = 0.07958;
+              $c = 0.007958;
             </pg-code>
           </setup>
           <statement>
             <p>Water flows onto a flat surface at a rate of 5cm<m>^3</m>/s forming a circular puddle 10mm deep. How fast is the radius growing when the radius is:</p>
 
             <ol>
-              <li><p>1 cm?</p></li>
+              <li><p>1 cm?</p>
+                <p><var name="$a" width="10"/></p>
+              </li>
 
-              <li><p>10 cm?</p></li>
+              <li><p>10 cm?</p>
+                <p><var name="$b" width="10"/></p>
+              </li>
 
-              <li><p>100 cm?</p></li>
+              <li><p>100 cm?</p>
+                <p><var name="$c" width="10"/></p>
+              </li>
             </ol>
           </statement>
-          <solution>
-            <ol>
-              <li><p><m>5/(2\pi) \approx 0.796</m>cm/s</p></li>
-
-              <li><p><m>1/(4\pi)\approx 0.0796</m> cm/s</p></li>
-
-              <li><p><m>1/(40\pi)\approx 0.00796</m> cm/s</p></li>
-            </ol>
-          </solution>
         </webwork>
       </exercise>
+<!--Exercise 4: Spherical balloon radius increasing-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$a">
+              <static>0.7958</static>
+            </var>
+            <var name="$b">
+              <static>0.007958</static>
+            </var>
+            <var name="$c">
+              <!--FUNGUS: The decimal 0.00007958 is not recognized as correct when I test it,
```

Comment:
There is no automatic way to compute a limit unless you do something like evaluate the expression at x=0.00000001 or something like that. Which I don't advise, because machine rounding could screw things up. I think you do the calculus off on some side paper calculating the exact value of the limit and then you define the value with that.